src/pages/assignments/components/assignment-card.vue

```diff
@@ -214,13 +224,57 @@ q-card.assignment(v-if="isFiltered && ((isExpired && history) || (!isExpired &&
         )
           q-item-section(style="max-width: 20px;")
             q-icon(name="fas fa-pencil-alt" size="14px")
-          q-item-section Edit
+          q-item-section Modify
+        q-item(
+          v-if="account === assignee"
+          clickable
+        )
+          q-popup-proxy(@before-show="newCommit = commited")
+            .confirm.column.q-pa-sm(style="max-width: 275px; background:white;")
+              | If you adjust your assignment, your % commitment will be immediately
+              | reflected on your next claim (no vote is necessary). Multiple adjustments
+              | during the same claim period will be included in the calculation.
+              .commit-group.q-mt-sm.q-px-sm.q-pb-sm(style="background: #EFEFEF;")
+                .row.q-pt-xs.q-pb-md New % Commitment
+                .row.q-mt-sm.q-px-md
+                  q-slider(
+                    v-model="newCommit"
+                    :min="0"
+                    :max="commited"
+                    :step="5"
+                    label
+                    :label-value="newCommit + '%'"
+                    label-always
+                    color="primary"
+                  )
+                .row.text-caption.justify-between
+                  span 0%
+                  span {{ commited + '%' }}
+              .row.flex.justify-between.q-mt-sm
+                q-btn(
+                  color="primary"
+                  label="No"
+                  dense
+                  flat
+                  v-close-popup="-1"
```

Comment:
v-close-popup is enough I think
Is there any reason why you put `="-1"` ?

Reply:
This was copied from other existing entries in the file. Quasar says it is needed to close a chain of popups and in this is situation, I think that's correct.

https://quasar.dev/vue-directives/close-popup#Closing-multiple-levels

Reply:
Ok I see I forgot about this